monkey/monkey_island/cc/resources/telemetry.py

```diff
@@ -188,7 +191,7 @@ def process_system_info_telemetry(telemetry_json):
             Telemetry.add_system_info_creds_to_config(creds)
             Telemetry.replace_user_dot_with_comma(creds)
         if 'mimikatz' in telemetry_json['data']:
-            users_secrets = mimikatz_utils.MimikatzSecrets.\
+            users_secrets = mimikatz_utils.MimikatzSecrets. \
```

Comment:
does it still work?

Reply:
Yeah. It's just a PEP8 automatic fix that PyCharm did for me